build/media-suite_helper.sh

```diff
@@ -336,27 +277,27 @@ do_vcs() {
         rm -f ./build_successful{32,64}bit{,_*}
         if [[ $build32$build64$bits == yesyes64bit ]]; then
             new_updates="yes"
-            new_updates_packages="$new_updates_packages [$vcsFolder]"
+            new_updates_packages="$new_updates    oldHead=$(vcs_get_current_head "$PWD")_packages [$vcsFolder]"
```

Comment:
Is this correct?

Reply:
uhhh, nope I'm pretty sure

Reply:
fixed, I think this was a copy paste issue